Lib/test/_test_multiprocessing.py

```diff
@@ -4724,6 +4724,40 @@ def test_level(self):
         root_logger.setLevel(root_level)
         logger.setLevel(level=LOG_LEVEL)
 
+    def assert_log_lines(self, expected_values, stream=None):
+        stream = stream or self.stream
+        actual_lines = stream.getvalue().splitlines()
+        self.assertEqual(len(actual_lines), len(expected_values))
+        for actual, expected in zip(actual_lines, expected_values):
+            self.assertEqual(actual, expected)
+        s = stream.read()
+        if s:
+            self.fail("Remaining output at end of log stream:\n" + s)
+
+    def test_filename(self):
+        logger = multiprocessing.get_logger()
+        default_level = logger.level
```

Comment:
I think `original_level` might be a more proper name for this. It does not have to be the default level.